catalog/pipeline/fbc-release/0.23/fbc-release.yaml

```diff
@@ -0,0 +1,393 @@
+      default: "true"
+  workspaces:
+    - name: release-workspace
+  results:
+    - name: requestMessage
+      type: string
+      value: $(tasks.add-fbc-contribution-to-index-image.results.requestMessage)
+    - name: requestReason
+      type: string
+      value: $(tasks.add-fbc-contribution-to-index-image.results.requestReason)
+    - name: iibIndexImage
+      type: string
+      value: $(tasks.extract-index-image.results.indexImage)
+    - name: iibIndexImageResolved
+      type: string
+      value: $(tasks.extract-index-image.results.indexImageResolved)
+  tasks:
+    - name: collect-data
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-bundles.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: catalog/task/collect-data/0.4/collect-data.yaml
+      params:
+        - name: release
+          value: $(params.release)
+        - name: releaseplan
+          value: $(params.releaseplan)
+        - name: releaseplanadmission
+          value: $(params.releaseplanadmission)
+        - name: releasestrategy
+          value: $(params.releasestrategy)
+        - name: snapshot
+          value: $(params.snapshot)
+        - name: subdirectory
+          value: $(context.pipelineRun.uid)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+    - name: verify-enterprise-contract
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: $(params.verify_ec_task_git_url)
+          - name: revision
+            value: $(params.verify_ec_task_git_revision)
+          - name: pathInRepo
+            value: $(params.verify_ec_task_git_pathInRepo)
+      params:
+        - name: IMAGES
+          value: "$(workspaces.data.path)/$(context.pipelineRun.uid)/snapshot_spec.json"
+        - name: SSL_CERT_DIR
+          value: /var/run/secrets/kubernetes.io/serviceaccount
+        - name: POLICY_CONFIGURATION
+          value: $(params.enterpriseContractPolicy)
+        - name: STRICT
+          value: "1"
+        - name: IGNORE_REKOR
+          value: "true"
+        - name: PUBLIC_KEY
+          value: $(params.enterpriseContractPublicKey)
+      workspaces:
+        - name: data
+          workspace: release-workspace
+      runAfter:
+        - collect-data
+    - name: validate-single-component
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/redhat-appstudio/release-service-bundles.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: catalog/task/validate-single-component/0.1/validate-single-component.yaml
+      params:
+        - name: snapshotPath
+          value: "$(context.pipelineRun.uid)/snapshot_spec.json"
+      runAfter:
+        - verify-enterprise-contract
```

Comment:
Wouldn't it make sense to run this check before EC? EC is much heavier and this is just a simple check, so I think it would make sense to start with that.